client/client-multi/client-c/src/platform-impl/EconaisEC19D/logger.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * econais_logger.c
+ *
+ *  Created on: Dec 31, 2014
+ *      Author: avp
+ */
+
+
+#include <sndc_sdk_api.h>
+#include "../../platform/system_logger.h"
+
+
+
+void write_log(FILE * sink, const char * buffer, size_t message_size) {
```

Comment:
Inappropriate code style and formatting. Look around the C SDK project for reference.